communitiesuk/funding-service-design-fund-store
# Fs 2910 ns data update

### Change description
Populating outstanding night shelter data: privacy notice and prospectus link

- [n/a] README and other documentation has been updated / added (if needed)


### How to test
Execute the script FS2910_ns_links.py to update the data
Then navigate to the start page and the prospectus and privacy links should work
